2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,12 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let report = [];
+    for(element of cities){
+        reportString = `The temperature in ${element} is ${temperatureService(element)} degrees`;
+        report.push(reportString);
+        };
+return report;
 }
 
```

Comment:
Nice job! small point: to properly declare variable, you should use let to declare element
.